Sources/Core/AWSSDKIdentity/Tests/AWSSDKIdentityTests/AWSCredentialIdentityResolverTests/ProcessAWSCredentialIdentityResolverTests.swift

```diff
@@ -9,9 +9,6 @@ import XCTest
 import struct AWSSDKIdentity.ProcessAWSCredentialIdentityResolver
 @_spi(FileBasedConfig) @testable import AWSClientRuntime
 
-// Test fails on CI build with macos-11, Xcode_13.2.1, platform=iOS Simulator but not on later versions
-// ProcessCredentialsProvider is not useful on iOS platform so this test will remain disabled for now
-#if !os(iOS)
```

Comment:
This conditional compile is no longer needed since the affected Xcode version is no longer supported.